src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java

```diff
@@ -101,6 +101,27 @@ public void findByZipFileName_should_find_envelops_in_db() {
         assertThat(resultForX).hasSize(0);
     }
 
+    @Test
+    public void findByCcdId_should_find_envelops_in_db() {
```

Comment:
naming is sth like this should_{expected}_when_{condition}

Reply:
done